pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/forward/FixedByteChunkSVForwardIndexTest.java

```diff
@@ -185,6 +185,8 @@ public void testLong(ChunkCompressionType compressionType, int version)
         Assert.assertFalse(fourByteOffsetReader.isFixedOffsetMappingType());
         Assert.assertFalse(eightByteOffsetReader.isFixedOffsetMappingType());
       }
+      eightByteOffsetReader.close();
```

Comment:
These are not required because they are already closed with try-with-resources call. Same for other places